### DISCUSSION TITLE
Fine Tuning For Graduation Project

### DISCUSSION THREAD

**Discussion by SaifBattah** [aub-mind/arabert#199] [Q&A]
Hello Anton! 
we are a group of students that are working to create a fine tuned model from arabert model.
our first step is to train the model in answering questions or making conversation between customer and the employee who receive orders.
this is a sample of the data 
![sample](https://github.com/aub-mind/arabert/assets/125483984/4e9b66b6-4406-4e4e-b021-49512df78fd0)

so what class should we fine tune? is it Question answering or what?
and how can we arrange the data to be fine tuned.
we will be very grateful to your assistance and help.
thanks

**Comment by SaifBattah** [comment]
@WissamAntoun

**Comment by WissamAntoun** [comment]
Hey,

AraBERT models can't generate text, you'll need to use the AraGPT2 models. But considering the trend that we are seeing, the models aren't big enough to handle such hard task. I think you can try something like Arabic LLaMA (AceGPT) or JAIS

**Reply by Boodyahmedhamdy** [reply]
@WissamAntoun 
### How to fine tune JAIS? 
I made a lot of search but found nothing useful, can you help please

**Comment by SaifBattah** [comment]
@WissamAntoun , thanks for your feedback, but we now want to proceed with question answering, how can we do the fine tuning (Code, Dataset sample) and if there is any IPYNB that can help us.
